.gitignore

```diff
@@ -9,3 +9,4 @@
 /tmp/
 *.gem
 /.idea/
+.ruby-version
```

Comment:
I added this because it makes more sense to create a matrix of versions of Ruby to test against, instead of having a single version. As such, let developers use whichever version (within that spectrum)